### PR TITLE
implemented commits linter

### PR DESCRIPTION
**Pull request by pdabrowsky**
https://app.asana.com/0/1202705605335915/1202705605335925/f

### CODE REVIEW

**Review by dominikx96 — CHANGES_REQUESTED** [review]
Partially ok, but a couple of things are missing:
- husky
- lint-staged

read more about that: https://laurieontech.com/posts/husky/

and elaborate here why we want to implement that in this way :)

**Review by dominikx96 — CHANGES_REQUESTED** [review]
Also, husky and commitlint don't work.

File: package.json [hunk 1]
```diff
@@ -33,5 +36,19 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "lint-staged": {
+    "*": [
+      "eslint",
```
[the remark this answers is not in the document]
Comment (pdabrowsky) [reply package.json:42]: `--fix` flag automatically fix problems

**Review by dominikx96 — CHANGES_REQUESTED** [review]
Please write down short explanation how the 'lint-staged' work

**Comment by pdabrowsky** [comment]
Anytime a js file is staged for a commit, lint-staged runs the ESLint linter with the:

-  `--cache` tells ESLint to only check changed files
-  `--fix` tells ESLint to automatically fix problems when possible

**Comment by pdabrowsky** [comment]
Commitlint is a simple tool that lints commit messages and makes sure they follow a set of rules.

**Review by dominikx96 — CHANGES_REQUESTED** [review]
Doesn't work for me, once I try to commit with a valid structure, I am getting: 
Screenshot 2022-08-09 at 18 41 34

**Pull request merged**